kore/src/Kore/Attribute/Subsort.hs

```diff
@@ -1,10 +1,14 @@
 {-|
-Module      : Data.Attribute.Subsort
-Description : Representation and parser for subsort attribute
 Copyright   : (c) Runtime Verification, 2018
 License     : NCSA
 -}
-module Kore.Attribute.Subsort where
+module Kore.Attribute.Subsort
```

Comment:
This seems like a common theme, so I'll ask even if the PR is not ready: Why would one switch to exporting everything or importing everything?

Reply:
This is switching to exporting a specific list, rather than everything. I think Tom is fixing this (due to adding `-Wmissing-export-lists`). This helps with not accidentally exporting a helper function when doing a refactoring, for example

Reply:
`-Wmissing-export-lists` requires every module to have an export list. The most useful thing about export lists, in my opinion, is that the compiler will warn you when you have unused code. This may not be a big problem, but I did find a few unused functions this way.